Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,510 @@
+
+    \textit{Projected Duration: 3 weeks}
+
+\subsubsection{Final Report}
+\label{subsubsec: final report}
+    At the end of this project a final report will be made to report the outcome. It will outline the design process, fabrication process, and testing process.
+    It will also include the nozzle theory used in the calculations along with all the relevant assumptions and exterior information as needed. The report will provide
+    all the test data and data analysis results. The final product for the report will be the discussion of the polynomial curve-fit. This will be an extremely useful tool
+    for future similar cold gas propulsion systems and must be discussed in depth with an example of how it will be used.
+
+    Once this paper is complete it will be given to the advisors of the SPEX research group for review and then further publications will be discussed. The paper would
+    be great for job interviews to display the work you have done in a formal manner, and one that the interviewer can refer back to at a later date.
+
+    \textit{Projected Duration: 2 weeks}
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+
+Creating a CGT will require a few skills, many of which can be learned throughout the project. The design process will require knowledge of systems and how they
+work together, basic nozzle theory, computer-aided design (CAD), basic electrical engineering, basic coding abilities, and machining. The majors that would
+be most qualified for satisifing these requirements are mechanical engineering \& technology and electrical engineering \& technology. But this project is certainly
+open to all majors because most of these skills can be taught in at least a basic manner.
+
+The basic nozzle theory required here can be adequately learned by reading the introductory chapters of "Rocket Propulsion Elements" by Sutton. Chapter 1 through 4 will
+provide a sufficient basis of understanding for this project.
+
+ The CAD software of choice is Solidworks. It is very common for industry and has all the features that the team
+will need throughout the process. The education edition also includes the simulation premium package that can allow the team to run flow simulations if needed. The procedure for
+doing this is complicated but can be learned to a basic degree on youtube.
+
+The basic electrical engineering including setting up and arduino or raspberry pi to read out the data from the load cell and turning the solenoid on and off. It
+should not be difficult for a first year or second year EE or EET to set this up. If it proves to be difficult, the team can each out to older students or professors for assistance.
+Dr. Patru would certainly be a great resource for this kind of help.
```

Comment:
Periods used for abbreviations have different spacing afterward than periods ending sentences. 
Add an interword space after all instances of "Dr.".
Example
```tex
Dr.\ Patru
```